<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="31" started-at="2013-06-10T10:28:46Z" finished-at="2013-06-10T10:28:46Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="31" started-at="2013-06-10T10:28:46Z" finished-at="2013-06-10T10:28:46Z">
      <class name="com.sirma.itt.javacoure.singleton.ObserverTest">
        <test-method status="PASS" signature="fillTestDate()[pri:0, instance:com.sirma.itt.javacoure.singleton.ObserverTest@d4694f]" name="fillTestDate" is-config="true" duration-ms="0" started-at="2013-06-10T13:28:46Z" finished-at="2013-06-10T13:28:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fillTestDate -->
        <test-method status="FAIL" signature="add(com.sirma.itt.javacourse.observer.TV)[pri:0, instance:com.sirma.itt.javacoure.singleton.ObserverTest@d4694f]" name="add" duration-ms="16" started-at="2013-06-10T13:28:46Z" data-provider="validData" finished-at="2013-06-10T13:28:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{=com.sirma.itt.javacourse.observer.TV@1d6c275}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[argument type mismatch]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: argument type mismatch
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add -->
      </class> <!-- com.sirma.itt.javacoure.singleton.ObserverTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
